ci/test/pipeline.template.yml

```diff
@@ -44,7 +44,7 @@ steps:
         depends_on: []
         priority: 1
         timeout_in_minutes: 60
-        branches: "main v*.*"
+        # branches: "main v*.*"
```

Comment:
Stray?

Reply:
I'll delete this before merging, this just makes sure that we run the `Build aarch64` job, whereas otherwise it wouldn't run until merging to `main`